src/scrobbler/listenbrainzscrobbler.cpp

```diff
@@ -429,7 +433,7 @@ void ListenBrainzScrobbler::UpdateNowPlaying(const Song &song) {
   title = title.remove(Song::kTitleRemoveMisc);
 
   QJsonObject object_track_metadata;
-  if (song.albumartist().isEmpty() || song.albumartist().compare(Song::kVariousArtists, Qt::CaseInsensitive) == 0) {
```

Comment:
albumartist can be empty, so artist needs to be used for artist_name in that case.

Reply:
Right, I'm not even sure why I removed this check...